avocado/core/xunit.py

```diff
@@ -199,9 +199,7 @@ def end_test(self, state):
             self.xml.add_skip(state)
         elif status == 'FAIL':
             self.xml.add_failure(state)
-        elif status == 'ERROR':
-            self.xml.add_error(state)
-        elif status == 'INTERRUPTED':
+        else:   # ERROR, INTERRUPTED, ...
```

Comment:
^^ I mean the new "PASS" mapping could be another variant of SKIP. By using "mapping" we might miss that completely and keep reporting it as PASS.